learn/experimental/autobatching.md

```diff
@@ -0,0 +1,42 @@
+# Auto-batching
+
+If enabled, multiple document addition requests will be automatically combined into a batch and processed together to speed up indexation.
```

Comment:
Might be me being dumb, but I read this and immediately think "If what is enabled?" Perhaps adding something like "If this feature is enabled?"